### PR TITLE
tls 1.3: editorial cleanup + specify protocol ID

### PR DESCRIPTION
**Pull request by raulk**

### CODE REVIEW

File: tls/tls.md [hunk 1]
```diff
@@ -42,16 +42,25 @@ The libp2p handshake uses TLS 1.3 (and higher). Endpoints MUST NOT negotiate low
 
 During the handshake, peers authenticate each other’s identity as described in [Peer Authentication](#peer-authentication). Endpoints MUST verify the peer's identity. Specifically, this means that servers MUST require client authentication during the TLS handshake, and MUST abort a connection attempt if the client fails to provide the requested authentication information.
 
+When negotiating the usage of this handshake dynamically, via a protocol agreement mechanism like [multistream-select 1.0](https://github.com/libp2p/specs/blob/master/connections/README.md#multistream-select), it MUST be identified with the following protocol ID:
```
Comment (marten-seemann) [inline tls/tls.md]: Better: it is identified. We're just giving it a name here, no need for normative language.

Comment (raulk) [reply tls/tls.md]: Needs to be normative for other implementations to comply, unless I'm missing something?

Comment (marten-seemann) [reply tls/tls.md]: I don't think the assignment of a code point requires normative language. But keep it that way if you prefer.

File: tls/tls.md [hunk 2]
```diff
@@ -63,9 +72,9 @@ In order to prove ownership of its host key, an endpoint sends two values:
...
+The peer signs the concatenation of the string `libp2p-tls-handshake:` and the public key that it used to generate the certificate carrying the libp2p Public Key Extension, using its private host key. This signature provides cryptographic proof that the peer was in possession of the private host key at the time the certificate was signed. Peers MUST verify the signature, and abort the connection attempt if signature verification fails.
 
-The public host key and the signature are ANS.1-encoded into the SignedKey data structure, which is carried in the libp2p Public Key Extension. The libp2p Public Key Extension is a x509 extension with the Object Identier 1.3.6.1.4.1.53594.1.1.
+The public host key and the signature are ANS.1-encoded into the SignedKey data structure, which is carried in the libp2p Public Key Extension. The libp2p Public Key Extension is a X.509 extension with the Object Identier `1.3.6.1.4.1.53594.1.1`, [allocated by IANA to the libp2p project at Protocol Labs](https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers).
```
Comment (MarkusTeufelberger) [inline tls/tls.md]: ANS.1 --> ASN.1

Identier --> Identifier

**Pull request merged**